internal/pkg/otel/samples/linux/gateway.yml

```diff
@@ -0,0 +1,101 @@
+receivers:
+  # Receives data from other Collectors in Agent mode
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317 # Listen on all interfaces
+      http:
+        endpoint: 0.0.0.0:4318 # Listen on all interfaces
+
+connectors:
+  # Routing connector splits infra metrics from other metrics and routes them to ECS-based pipeline
+  # Other metrics are exported in OTel-native format to Elasticsearch
+  routing:
+    default_pipelines: [metrics/otel]
+    error_mode: ignore
+    table:
+      - context: metric
+        statement: route() where instrumentation_scope.name == "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver" or IsMatch(instrumentation_scope.name, "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/*")
+        pipelines: [metrics/infra/ecs, metrics/otel]
+  elasticapm: {} # Elastic APM Connector
+
+processors:
+  elasticinframetrics:
+    add_system_metrics: true
+    add_k8s_metrics: true
```

Comment:
Do we need these lines for this use case?

Reply:
Apart from these lines, everything else looks good to me btw 👍

Reply:
Probably not, but then why were they in the Gateway config for k8s? 🤔 @AlexanderWert do you know?